### DISCUSSION TITLE
Pastas Dagen 2023 + Discussie

### DISCUSSION THREAD

**Discussion by martinvonk** [pastas/pastas#438] [Courses&Events]
# Aankondiging

**(Dutch only)

Op 8 en 9 februari 2023 organiseren we de Pastas Dagen! Het doel van deze dagen is om Pastas gebruikers en ontwikkelaars met elkaar in contact te brengen en ervaringen uit te wisselen over het gebruik van Pastas. De ontwikkelaars zullen de nieuwste ontwikkelingen delen, waaronder de nieuwste Pastas Versie 1.0 Arrabiata, en de Pastastore. Daarnaast zullen verschillende gebruikers case-studies presenteren en is er gedurende de dagen veel ruimte voor discussies en brainstorms over het verder ontwikkelen van Pastas.

### Pastas cursus -  8 Februari
Op 8 februari van de Pastas Dagen wordt een cursus georganiseerd. Deze cursus is bedoeld voor beginnende en licht gevorderde gebruikers. Hierbij wisselen we af tussen presentaties en oefenen in Jupyter Notebooks om zo een interactieve dag te organiseren. Daarnaast willen we aan het eind van de cursus ook met eigen tijdreeksen aan de slag. Voor de cursus is er alleen een laptop (en oplader nodig).

### Pastas gebruikers dag - 9 Februari
Op 9 februari organiseren we de gebruikersdag. Deze staat in het teken van het delen van ervaringen en de laatste ontwikkelingen rondom Pastas. Zo presenteren we de nieuwste features van Pastas 1.0, voorbeelden uit de laatste wetenschappelijke artikelen en is er een workshop voor een zo efficiënt mogelijke workflow voor pastas. Gedurende de dag zijn er verschillende mogelijkheden voor gebruikers om input leveren over dingen waar zij tegenaan lopen met Pastas en waar zij graag nieuwe ontwikkelingen in zien. Er zijn ook een aantal gebruikers die hun ervaring en toepassing met Pastas presenteren:

- Pytrik Graafstra (Aveco de Bondt) over de toepassing van Pastas bij een ingenieursbureau;
- Anouk Sprong (Vitens) over de effectbepaling van winningen met tijdreeksanalyse in een geologisch complex systeem;
- Tim Franken en Jorn van de Velde (Sumaqua) over de toepassing van tijdreeksanalyse in België;
- Marjolein van Huijgevoort (KWR) over droogte analyse in Nederland.

#### Inschrijven kan tot uiterlijk 6 februari 12:00
Inschrijven kan via https://bit.ly/pastas_dagen. De kosten voor de cursus bedragen €500,- en voor de gebruikersdag €125,- inclusief BTW. Beide dagen zijn inclusief koffie en lunch en vinden plaats op de TU Delft faculteit CiTG.

Het Pastas Ontwikkelaarsteam
Raoul Collenteur (Eawag)
Mark Bakker (TU-Delft)
Frans Schaars (Artesia)
Ruben Caljé (Artesia)
Davíd Brakenhoff (Artesia)
Onno Ebbens (Artesia)
Martin Vonk (Artesia & TU-Delft)

![Pastas Dagen 2023](https://user-images.githubusercontent.com/66305055/205297294-9700bd4f-b24e-40b8-8fa5-1a98e36032b5.png)

**Comment by martinvonk** [comment]
## Cursusdag
Het voorlopige programma voor de cursus op 8 februari ziet er als volgt uit:

### Voorlopig Programma 
| TIJD | ACTIVITEIT                                             |
|------|--------------------------------------------------------|
| 0830 | Inloop                                                 |
| 0900 | Presentatie Tijdreeksanalyse met Impulsresponsfuncties |
| 0945 | Oefenen Jupyter Notebooks                              |
| 1030 | Presentatie Kiezen van de Modelstructuur               |
| 1115 | Oefenen Jupyter Notebooks                              |
| 1200 | Lunch                                                  |
| 1300 | Presentatie Kalibratie en Beoordeling van het Model    |
| 1400 | Oefenen Jupyter Notebooks (evt. met eigen reeksen)     |
| 1530 | Presentatie Afsluitend                                 |
| 1600 | Einde                                                  |

### Leerdoelen
Aan het eind van de cursus kan je:
* het basis principe van tijdreeksanalyse met impuls respons functies begrijpen
* een tijdreeksmodel maken in Pastas met:
  - verschillende respons functies
  - meerdere stresses
  - niet-lineariteit
 * een model beoordelen op basis van verschillende statistieken
 * zoeken binnen de documentatie van Pastas om de mogelijkheden te doorzoeken
   - waaronder gebruik maken van GitHub om vragen te stellen / problemen aan te kaarten
 * een (eigen) stijghoogte tijdreeks modelleren met data van verschillende bronnen

**Reply by martinvonk** [reply]
Voor de cursisten met weinig ervaring met Python is het goed om op niveau te komen. Hiervoor is het materiaal van [Exploratory computing with Python](https://mbakker7.github.io/exploratory_computing_with_python/) van Mark Bakker zeer geschikt. Op de website staan Jupyter Notebooks en video's die de Python programmeertaal duidelijk uitleggen. Hierdoor blijft er tijdens de cursus zoveel mogelijk tijd over om te leren over tijdreeksanalyse en aan de slag te gaan met Pastas.

Voor de cursus zijn de volgende notebooks en video's het belangrijkst:

- Basics and Plotting: [Notebook 1](http://nbviewer.ipython.org/github/mbakker7/exploratory_computing_with_python/blob/master/notebook1_basics_plotting/py_exploratory_comp_1_sol.ipynb) - [Video](https://youtube.com/watch?v=a505WqfH5Tg&si=EnSIkaIECMiOmarE)
- Arrays: [Notebook 2](http://nbviewer.ipython.org/github/mbakker7/exploratory_computing_with_python/blob/master/notebook2_arrays/py_exploratory_comp_2_sol.ipynb) - [Video](https://youtu.be/5RkeHZnZEnM)
- For loops and If/Else statements: [Notebook 3](http://nbviewer.ipython.org/github/mbakker7/exploratory_computing_with_python/blob/master/notebook3_for_and_if/py_exploratory_comp_3_sol.ipynb) - [Video](https://youtu.be/19gM-QEVugc)
- Functions: [Notebook 4](http://nbviewer.ipython.org/github/mbakker7/exploratory_computing_with_python/blob/master/notebook4_functions/py_exploratory_comp_4_sol.ipynb) - [Video](https://youtu.be/ZqjYNtWanMM)
- Pandas and Time Series: [Notebook 8](http://nbviewer.ipython.org/github/mbakker7/exploratory_computing_with_python/blob/master/notebook8_pandas/py_exploratory_comp_8_sol.ipynb) - [Video](https://youtu.be/MTdIY6uFY6M)
- Object oriented programming: [Notebook 12](http://nbviewer.ipython.org/github/mbakker7/exploratory_computing_with_python/blob/master/notebook12_oop/py_exploratory_comp_12_sol.ipynb) - [Video](https://youtu.be/pNLAEDbK03s)

**Comment by martinvonk** [comment]
## Gebruikersdag
Het voorlopige programma voor de gebruikersdag op 9 februari ziet er als volgt uit:

### Voorlopig Programma 
| TIJD  | ACTIVITEIT                                                                                     |
|-------|------------------------------------------------------------------------------------------------|
|  9:00 | Inloop                                                                                         |
|  9:30 | Welkomstpraatje                                                                  |
|  9:45 | Presentatie Geschiedenis Tijdreeksanalyse & Open-Source                                        |
| 10:00 | Presentatie Pastas 1.0 Arrabiata                                                               |
| 10:30 | Koffiepauze                                                                                    |
| 11:00 | Presentatie Keuze Modelstructuur                                                           |
| 11:30 | Presentatie Toepassing Inschatten Verlaging van Verschillende Pompstations                     |
| 12:00 | Lunch                                                                                          |
| 13:00 | Presentaties Toepassingen                                                                      |
|          |   - Toepassing van Pastas bij een ingenieursbureau (Aveco de Bondt)                              |
|          |   - Effectbepaling van winningen met tijdreeksanalyse in een geologsich complex systeem (Vitens) |
|          |   - Tijdreeksanalyse in België (Sumaqua)                                                         |
|          |   - Droogte analyse in Nederland (KWR)                                                           |
| 14:30 | Presentatie Python Tools voor Nederlandse Tijdreeksen                                                      |
| 15:00 | Koffiepauze                                                                                    |
| 15:30 | Discussie Toekomst Pastas                                                                      |
| 16:30 | Borrel                                                                                         |

**Comment by raoulcollenteur** [comment]
Dear All,

Please find the presentation of Pastas 1.0 here:

[02_Pastas_Arrabiata_public.pdf](https://github.com/pastas/pastas/files/10736729/02_Pastas_Arrabiata_public.pdf)

Thanks for the great day(s) last week!

Cheers,

Raoul

**Comment by martinvonk** [comment]
Toekomst Pastas Discussie Flipovers:

[Pastast Dagen Discussie.pdf](https://github.com/pastas/pastas/files/10741100/Pastast.Dagen.Discussie.pdf)

**Comment by dbrakenhoff** [comment]
Dear all, 

I've linked the presentation on timeseries analysis with multiple well fields (tijdreeksanalyse met meerdere onttrekkingen) below:

[04_pastas-gebruikersdag_tijdreeksanalyse-met-meerdere-onttrekkingen_public.pdf](https://github.com/pastas/pastas/files/10741328/04_pastas-gebruikersdag_tijdreeksanalyse-met-meerdere-onttrekkingen_public.pdf)

Cheers,

Davíd

**Comment by OnnoEbbens** [comment]
Hierbij nog de presentatie en notebook die ik heb laten zien. Beide in pdf format.

[Hydropandas en pastastore.pdf](https://github.com/pastas/pastas/files/10753095/Hydropandas.en.pastastore.pdf)
[Voorbeeld.pdf](https://github.com/pastas/pastas/files/10753096/Voorbeeld.pdf)
